src/lib/pinata/pinata.ts

```diff
@@ -198,7 +208,13 @@ export default function usePinata() {
       metadata: metadataFilter,
     };
     
-    const pinataResult = (await pinata.pinList(filters));
+    var pinataResult: PinataPinListResponse = {count: 0, rows: []};
+
+    try{
+      pinataResult = await pinata.pinList(filters)
+    } catch(err){
+      console.log("Error retrieving open tickets from pinata: ", err);
+    }
 
     return pinataResult
```

Comment:
move the return into the try block?

Reply:
done; also adjusted other functions using portions of the result